lib/Navigation/Renderer/Menu.php

```diff
@@ -697,7 +697,7 @@ protected function _renderDeepestMenu(
      * Renders a normal menu (called from {@link renderMenu()})
      *
      * @param  Container $container     container to render
-     * @param  string                    $ulClass       CSS class for first UL
+     * @param  string                    $ulClasses       CSS class for UL levels
```

Comment:
`$ulClasses` can now be both `string` and `array`. Annotation `@param  string|string[]` would be more appropriate.